app/services/data_migrations/backfill_setup_interviews_permission.rb

```diff
@@ -0,0 +1,12 @@
+module DataMigrations
+  class BackfillSetupInterviewsPermission
+    TIMESTAMP = 20210701153532
+    MANUAL_RUN = false
```

Comment:
We want to trigger this manually as soon as the flag is enabled 
```suggestion
    MANUAL_RUN = true
```

Reply:
Cant believe i missed that one =P